File: c/Ex11.c
#include <stdio.h>
 
int main(int argc, char *argv[])
{
   float n1, n2; 
  printf("Digite o valor do primeiro numero: ");
  scanf("%f", &n1);
  printf("Digite o valor do segundo numero: ");
  scanf("%f", &n2);
  if (n1 > n2)
  {
      printf("o numero maioir e %.2f", n1);
  }
 else if(n1==n2)
  {
      printf("numeros iguais");
  }
  else
  {
      printf("o numero maioir e %.2f", n2);
  }
 
  return 0;
}

File: python/Ex11.py
n1 = float(input("Digite o primeiro numero: "))
n2 = float(input("Digite o segundo numero: "))
if(n1==n2):
    print(f"Os dois numeros tem o mesmo valor:{n1: .2f}")
else:
    if(n1 > n2):
        print(f"maior valor é:{n1: .2f}")
    else:
        print(f"maior valor é:{n2: .2f}")